CRM/Mailing/BAO/Mailing.php

```diff
@@ -816,10 +816,8 @@ private function _getTokens($prop) {
    *
    * @param array $testParams
    *   Contains form values.
-   *
-   * @return void
    */
-  public function getTestRecipients($testParams) {
+  public static function getTestRecipients(array $testParams, $mailingID): void {
```

Comment:
No universe hits - which makes sense as there is a supported api to get this info

![image](https://github.com/civicrm/civicrm-core/assets/336308/01b914cb-3a50-499e-85bf-2cab9cf2445e)